<launch>
  <!--Camera 1: measure the amount-->
  <node type="color_detector_node" pkg="lfd_vision" name="color_detector_node">
    <param name="camera" value="1" />
    <param name="mode" value="2" />
    <param name="num_detectors" value="1" />
    <param name="color_files_base" value="$(find lfd_vision)/x" />
  </node>

  <!--Camera 0: cup orientation tracker-->
  <node type="color_detector_node" pkg="lfd_vision" name="color_detector_node_h">
    <remap from="color_middle_xy" to="color_middle_xy1" />
    <remap from="color_occupied_ratio" to="color_occupied_ratio1" />
    <param name="camera" value="0" />
    <param name="mode" value="2" />
    <param name="num_detectors" value="2" />
    <param name="color_files_base" value="$(find lfd_vision)/h" />
  </node>

  <!--<node type="human_demo_logger.py" pkg="lfd_vision" name="human_demo_logger_node">
    <param name="file_prefix" value="" />
  </node>-->
</launch>
